src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -101,6 +108,13 @@ public virtual void WriteCsvLine(string line)
         /// </summary>
         protected override void BeginProcessing()
         {
+            if (this.MyInvocation.BoundParameters.ContainsKey(nameof(QuoteFields)) && this.MyInvocation.BoundParameters.ContainsKey(nameof(UseQuotes)))
```

Comment:
Any reason we don't put them in separate parameter sets?

Reply:
Complicity. We have base class with parameters and some derived classes adding parameters too - parameter sets turn into a trick.

Reply:
Got it. Thanks